docs/tools/bench.md

```diff
@@ -18,7 +18,7 @@ The current benchmarks are:
 * `ackermann`: Print the value of `Ack(m, n)`, the [two-argument Ackermann-Péter function][ackermann].
 * `sum-sq-diff`: Output the difference between the sum of the squares of the first *n* natural numbers and the square of their sum.
 * `sqrt`: Implements the [Newton-Raphson Method][newton] of approximating the square root of a number to arbitrary precision
-
+* `prefect`: Check if input argument is a pefect number.  Returns output as linux style return code. 
```

Comment:
"perfect" and "pefect" is misspelled twice. Also, capitalize "Linux." But maybe I'd call this Unix-style, because it's not really specific to Linux?

Reply:
I have incorporated the changes, please let me know if anything else needs changing.